docs/guides/atlassian-integration.md

```
@@ -65,6 +65,33 @@ export ATLASSIAN_REQUEST_DELAY=1000  # ミリ秒
 michi jira:sync my-feature
 ```
 
+### チケットタイトルフォーマット
+
+JIRAチケットのタイトルには、識別性を高めるために**リポジトリ名**と**機能名**が自動的にプレフィックスとして付与されます。
+
+**Epic のタイトルフォーマット**:
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)